### ISSUE TITLE
 Content Security Policy directive: "img-src 'self' data:".

### ISSUE THREAD

**Issue by Arpanx** [asadsahi/AspNetCoreSpa#163]
I can not upload pictures from another site.
I'm trying to redefine rules
`<meta http-equiv="Content-Security-Policy" content="img-src 'self' data:*"><!-- -->`
But the rules are still given as in the header.
Where do they come from, and how to allow downloading from other people's sites?

In the browser I get this error:
_Refused to load the image 'http://c.tile.openstreetmap.fr/hot/12/2397/1379.png' because it violates the following Content Security Policy directive: "img-src 'self' data:"._

**Comment by asadsahi** [comment]
@Arpanx CSP rules are defined [here](https://github.com/asadsahi/AspNetCoreSpa/blob/master/Server/Extensions/ApplicationBuilderExtensions.cs#L67) you can amend them as per your need. For more information see [this](https://joonasw.net/view/csp-in-aspnet-core)

**Comment by Arpanx** [comment]
Thank you. Everything is working!
Wonderful assembly, and a big piece of work you have done. Anyone who makes money on it, it's good to share it with the author. I'm just starting one small startup.
I ran this entirely on Linux Ubuntu + PostrgeSQL EF 
+ I had to change the version in this line, otherwise on Linux there was no restore of packages
`    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="**2.0.1**" PrivateAssets="All" />`
